app/src/main/java/org/fossasia/openevent/Services/BookmarkAlarmService.java

```diff
@@ -0,0 +1,68 @@
+package org.fossasia.openevent.Services;
+
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Intent;
+import android.media.RingtoneManager;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.activities.SessionDetailActivity;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.dbutils.DbSingleton;
+import org.fossasia.openevent.utils.IntentStrings;
+
+/**
+ * Created by Manan Wason on 21/08/15.
+ */
+public class BookmarkAlarmService extends Service {
```

Comment:
You should not extend from service instead you should use IntentService for this purpose.